prime-router/src/main/kotlin/history/azure/DatabaseDeliveryAccess.kt

```diff
@@ -91,15 +92,7 @@ class DatabaseDeliveryAccess(
         }
     }
 
-    /**
-     * Fetch the details of an action's relations (descendants).
-     * This is done through a recursive query on the report_lineage table.
-     *
-     * @param actionId the action id attached to the action to find relations for.
-     * @param klass the class that the found data will be converted to.
-     * @return a list of descendants for the given action id.
-     */
-    override fun <T> fetchRelatedActions(actionId: Long, klass: Class<T>): List<T> {
+    override fun <T> fetchRelatedActions(reportId: UUID, klass: Class<T>): List<T> {
```

Comment:
kdoc

Reply:
The kdoc gets inherited from the base class.